src/cljs/time_tracker_web_nxt/views.cljs

```diff
@@ -17,35 +17,40 @@
 (defn element-value [event]
   (-> event .-target .-value))
 
-(defn project-dropdown [projects selected]
-  (let [selected-id (:id @selected)]
-    [:select.project-dropdown
-     {:value (or selected-id "")
-      :on-change #(reset! selected {:id (-> % .-target .-value)})}
-     (for [{:keys [id name]} projects]
-       ^{:key id} [:option (if (= selected-id id)
-                             {:value id :label name}
-                             {:value id :label name}) name])]))
+(defn dropdown-widget [items selected-id event]
+  [:select.project-dropdown
```

Comment:
If you're creating a generic dropdown widget, do apply a generic class to the element.